starter-code/src/clue.js

```diff
@@ -1,94 +1,156 @@
+    last_name:    "Orchid",
+    color:        "white",
+    description:  "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+    age:          26,
+    image:        "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+    occupation:   "Scientist"
+  },
+  profPlum: {
+    first_name:   "Victor",
+    last_name:    "Plum",
+    color:        "purple",
+    description:  "Billionare video game designer",
+    age:          22,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+    occupation:   "Designer"
+  },
+  missScarlet: {
+    first_name:   "Kasandra",
+    last_name:    "Scarlet",
+    color:        "red",
+    description:  "She is an A-list movie star with a dark past",
+    age:          31,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+    occupation:   "Actor"
+  },
+  mrsPeacock: {
+    first_name:   "Eleanor",
+    last_name:    "Peacock",
+    color:        "blue",
+    description:  "She is from a wealthy family and uses her status and money to earn popularity",
+    age:          36,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation:   "Socialité"
+  },
+  mrMustard: {
+    first_name:   "Jack",
+    last_name:    "Mustard",
+    color:        "yellow",
+    description:  "He is a former football player who tries to get by on his former glory",
+    age:          62,
+    image:        "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+    occupation:   "Retired Football player"
+  }
+}
+
+let Weapons = [
+  {
+    name: "rope",
+    weight: 10
+  },
+  {
+    name: "knife",
+    weight: 8
+  },
+  {
+    name: "candlestick",
+    weight: 2
+  },
+  {
+    name: "dumbbell",
+    weight: 30
+  },
+  {
+    name: "poison",
+    weight: 2
+  },
+  {
+    name: "axe",
+    weight: 15
+  },
+  {
+    name: "bat",
+    weight: 13
+  },
+  { name: "trophy",
+    weight: 25
+  },
+  {
+    name: "pistol",
+    weight: 20
+  }
+]
+
+let Rooms = [
+  {name: "Dinning Room"},
+  {name: "Conservatory"},
+  {name: "Kitchen"},
+  {name: "Study"},
+  {name: "Library"},
+  {name: "Billiard Room"},
+  {name: "Lounge"},
+  {name: "Ballroom"},
+  {name: "Hall"},
+  {name: "Spa"},
+  {name: "Living Room"},
+  {name: "Observatory"},
+  {name: "Theater"},
+  {name: "Guest House"},
+  {name: "Patio"}
+]
+
+// Characters Array
+let charactersArray = Object.values(Characters);
+console.log(charactersArray);
+
+
+//CREATING MISTERY
+function randomSelector(array) {
+  let randomNum = Math.floor(Math.random() * array.length);
+  return array[randomNum];
+}
+console.log(randomSelector(Rooms));
+
+
+function pickMistery() {
+  let misteryEnvelope = [];
+  let charactersStack = misteryEnvelope.push(randomSelector(charactersArray));
+  let weaponsStack = misteryEnvelope.push(randomSelector(Weapons));
+  let roomStack = misteryEnvelope.push(randomSelector(Rooms));
+  return misteryEnvelope;
```

Comment:
you put 3 let's here for charactersStack, weaponsStack and roomStack. This is in itself not a bad idea but if you are not gonna use them you can leave them out:-)
Kido